x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/main.tsx

```diff
@@ -79,6 +169,45 @@ export const PipelinesList: React.FunctionComponent<RouteComponentProps> = ({
     redirectToPathOrRedirectPath(getListPath());
   };
 
+  const [filters, setFilters] = useState<Filters<FilterName>>({
+    managed: {
+      name: i18n.translate('xpack.idxMgmt.indexTemplatesList.viewManagedTemplateLabel', {
+        defaultMessage: 'Managed pipelines',
+      }),
+      checked: 'on',
+      handleFilter: (allPipelines: Pipeline[]) => isManagedPipelines(allPipelines, true),
+    },
+    notManaged: {
+      name: i18n.translate('xpack.idxMgmt.indexTemplatesList.viewNotManagedTemplateLabel', {
+        defaultMessage: 'Not managed pipelines',
```

Comment:
Let's change the copy here

```suggestion
        defaultMessage: 'Custom pipelines',
```